modules/cnv_vm_wizard_fields_web.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-vm-wizard-fields-web]]
+== virtual machine wizard fields
+
+|===
+|Name |Parameter |Description
+
+|Name
+|
+|Name of the virtual machine.
+
+|Description
+|
+|Optional description field.
+
+|Template
+|
+|Template from which to create the virtual machine. Selecting a template will automatically complete other fields. 
+
+
+.3+|Provision Source
+|PXE
+|Provision virtual machine from PXE menu. Requires a PXE-capable NIC in the cluster.
```

Comment:
Maybe worth mentioning that PXE capable nic actually means a normal nick but is associated to a network attachment definition which is backed by an L2 network. e.g. you need to have ovs-cni + multus set up and configured. Im sure it is documented somewhere else so you can link to that part from here.